docs/source/changelog.rst

```diff
@@ -10,8 +10,8 @@ All notable changes to this project will be documented in this file.
 
   Given a version number MAJOR.MINOR.HOTFIX:
 
-  - MAJOR version change when major text modifications are made (for example: new chapter, changing how a curriculum guide teaches a subject).
-  - MINOR version change when content or functionality is added or updated (for example: new videos, new activities, large number of text (typo/grammar) fixes).
+  - MAJOR version change when system-wide modifications incompatible with previous versions are made (for example: switching to a new database system).
```

Comment:
Could we change it from `database system` to `backend system` or `software solution` as Django allows you to switch databases with roughly one line of code.

Reply:
Done :+1: